ports/esp32/mphalport.c

```diff
@@ -75,7 +81,16 @@ void check_esp_err(esp_err_t code) {
             return;
         }
         o_str->base.type = &mp_type_str;
+        #if MICROPY_ERROR_REPORTING > MICROPY_ERROR_REPORTING_NORMAL
+        char err_msg[64];
+        esp_err_to_name_r(code, err_msg, sizeof(err_msg));
```

Comment:
Why can't you use `esp_err_to_name` like below?  Then it doesn't need a temporary buffer on the C stack.

Reply:
[Error Code and Helper Functions](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/esp_err.html#_CPPv417esp_err_to_name_r9esp_err_tPc6size_t)

` esp_err_to_name_r()` can return system error codes. It may be helpful for developers.
` esp_err_to_name()` is enough for users.

Reply:
I see.  Then please keep using this `esp_err_to_name_r` function.